src/coreclr/inc/bundle.h

```diff
@@ -18,29 +18,35 @@ class Bundle;
 struct BundleFileLocation
 {
     INT64 Size;
+    void* DataStart;
     INT64 Offset;
     INT64 UncompresedSize;
 
     BundleFileLocation()
-    { 
+    {
         LIMITED_METHOD_CONTRACT;
 
         Size = 0;
-        Offset = 0; 
+        DataStart = INVALID_HANDLE_VALUE;
```

Comment:
```suggestion
        DataStart = nullptr;
```
`INVALID_HANDLE_VALUE` is meant for HANDLEs. DataStart is a pointer. It is not a good idea to mix HANDLEs and pointers.